app/models/manageiq/providers/amazon/refresh_helper_methods.rb

```diff
@@ -41,6 +41,17 @@ def add_instance_disk(disks, size, name, location)
     super(disks, size, name, location, "amazon")
   end
 
+  def add_block_device_disk(disks, name, location)
+    disk = {
+      :device_name     => name,
+      :device_type     => "disk",
+      :controller_type => "amazon",
+      :location        => location,
+    }
+    disks << disk
```

Comment:
@bronaghs 
* I would also prefer `EBS volume` or something similar, so I'd be happy to use that instead of `amazon`. Honestly, instead of `amazon` for instance storage, I'd also prefer type that would explicitly say it's ephemeral storage, but I don't know currently what this would mean for other parts of the code.
* From the list of mappings I can only get the device (and the volume_id). Other data will be parsed by the EBS storage manager. #125 will populate the remaining data. This PR is basically preparing for the other because currently, cloud_manager deletes all links storage manager creates :(.
* linking is done in #125